app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
change line 2 and 3 to `def home; end`

Because this is `coding convention`. The method has not body, you can write it on once line.